kudu-handler/pom.xml

```diff
@@ -98,6 +98,15 @@
       <artifactId>junit-vintage-engine</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-1.2-api</artifactId>
```

Comment:
why is this change?

Reply:
These logging dependencies are required by upgraded kudu. There has been internal changes in kudu moving from 1.12.0 to 1.17.0 and that requires us to explicitly add these required logging dependencies.